File: abc/abc345/a.cpp
// https://atcoder.jp/contests/abc345/submissions/51362056

#include <bits/stdc++.h>
using namespace std;

int main() {
  string s;
  cin >> s;

  regex pattern("^<=+>$");
  cout << (regex_search(s, pattern) ? "Yes" : "No");
}

File: abc/abc345/a.java
// https://atcoder.jp/contests/abc345/submissions/51362183

import java.util.Scanner;
import java.util.regex.Matcher;
import java.util.regex.Pattern;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    Pattern p = Pattern.compile("^<=+>$");
    Matcher m = p.matcher(s);

    System.out.print(m.find() ? "Yes" : "No");
  }
}
